app/ui/legacy/src/main/res/layout/account_list_item.xml

```diff
@@ -1,49 +1,72 @@
+    tools:ignore="UnusedAttribute">
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="vertical"
-        android:paddingLeft="24dp"
-        android:paddingRight="16dp">
+        android:background="?android:attr/windowBackground"
+        android:clickable="true"
+        android:focusable="true"
+        android:foreground="?selectableItemBackground"
+        android:gravity="center_vertical"
+        android:orientation="horizontal"
+        android:paddingLeft="8dp">
 
-        <TextView
-            android:id="@+id/name"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:ellipsize="end"
-            android:paddingBottom="4dp"
-            android:paddingTop="8dp"
-            android:singleLine="true"
-            android:textAppearance="?android:attr/textAppearanceListItem"
-            tools:text="Personal" />
-
-        <TextView
-            android:id="@+id/email"
-            android:layout_width="wrap_content"
+        <ImageView
+            android:layout_width="40dp"
+            android:layout_height="40dp"
+            android:contentDescription="@string/account_settings_action"
+            android:padding="8dp"
+            app:srcCompat="?attr/iconSettingsAccount" />
+
+        <LinearLayout
+            android:layout_width="313dp"
             android:layout_height="wrap_content"
-            android:ellipsize="end"
-            android:paddingBottom="8dp"
-            android:singleLine="true"
-            android:textAppearance="?android:attr/textAppearanceSmall"
-            tools:text="test@example.org" />
+            android:orientation="vertical"
+            android:paddingLeft="24dp"
+            android:paddingRight="16dp">
+
+            <TextView
+                android:id="@+id/name"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:ellipsize="end"
+                android:fadeScrollbars="true"
+                android:paddingTop="8dp"
+                android:paddingBottom="4dp"
+                android:singleLine="true"
+                android:textAppearance="?android:attr/textAppearanceListItem"
+                tools:text="Personal" />
+
+            <TextView
+                android:id="@+id/email"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:ellipsize="end"
+                android:paddingBottom="8dp"
+                android:singleLine="true"
+                android:textAppearance="?android:attr/textAppearanceSmall"
+                tools:text="test@example.org" />
+
+
+        </LinearLayout>
+
+        <ImageView
+            android:id="@+id/drag_handle"
+            android:layout_width="56dp"
+            android:layout_height="56dp"
```

Comment:
To make this view take up the whole height (which we want because it's clickable) without deforming the image use the following:

```xml
android:scaleType="center"
android:layout_height="match_parent"
```